Formula/m/mdbtools.rb

```diff
@@ -28,10 +28,18 @@ class Mdbtools < Formula
   depends_on "glib"
   depends_on "readline"
 
+  on_macos do
+    depends_on "gettext"
+  end
+
+  on_linux do
+    depends_on "gettext" => :build
```

Comment:
You can just make this unconditional since macOS also depends on `gettext`.

Reply:
works for me 👍